animate/metric.py

```diff
@@ -150,6 +235,17 @@ def set_parameters(self, metric_parameters={}):
             raise NotImplementedError(
                 "Isotropic metric optimisations are not supported in Firedrake."
             )
+        if self._plex.metricRestrictAnisotropyFirst():
+            raise NotImplementedError(
+                "Restricting metric anisotropy first is not supported in Firedrake."
+            )
```

Comment:
This should be easy enough to implement (just tweak the arguments to the `enforce_spd` call in the definition of `normalise`. However, I'd prefer to address it in a separate issue rather than here, considering the impact with some tests.

Reply:
See #100.